src/cpg_flow/filetypes.py

```diff
@@ -228,7 +228,42 @@ def __repr__(self):
         )
 
 
-class FastqPairs(list[FastqPair], AlignmentInput):
+@dataclass
+class FastqOraPair(AlignmentInput):
+    def __init__(self, r1_path: FastqPath, r2_path: FastqPath, reference_assembly: str | Path):
+        self.reference_assembly = reference_assembly
+        self.fq_pair = FastqPair(r1_path, r2_path)
+
+    def as_resources(self, b) -> 'FastqPair':
+        """
+        Makes a trio of ResourceFile objects for r1, r2, and a reference assembly.
+        """
+        return b.read_input_group(
+            r1=self.fq_pair.r1,
+            r2=self.fq_pair.r2,
+            reference=self.reference_assembly,
+        )
+
+    def exists(self) -> bool:
+        """
+        Check if each FASTQ.ora file in the pair, and the reference, exists.
+        """
+        return all(exists(file) for file in [self.fq_pair.r1, self.fq_pair.r2, self.reference_assembly])
+
+    def __repr__(self):
+        """
+        Glob string to find all FASTQ files.
+
+        >>> str(FastqPair('gs://sequencing_group_R1.fq.gz', 'gs://sequencing_group_R2.fq.gz'))
```

Comment:
```suggestion
        >>> str(FastqOraPair('gs://sequencing_group_R1.fq.gz', 'gs://sequencing_group_R2.fq.gz'))
```

Reply:
fixed! Sorry I didn't accept this comment exactly, there were a few other tweaks (`.gz` -> `.ora` in the examples)